### PR TITLE
Update leaderboard scores list using data from API

### PR DESCRIPTION
**Pull request by PNdunguMaina**
In this project, I did the following: :writing_hand: 

- [ ] Utilized the provided API documentation to consume the API
- [ ] Created a new game using the provided API
- [ ] Implemented the "Refresh" button to receive data from the API and destringify it
- [ ] Implemented the "Submit" button to send data to the API
- [ ] Used async and await Javascript features to consume the API
- [ ] Followed Gitflow
- [ ] Used Webpack to bundle Javascript
- [ ] Used ES6 modules to write modular Javascript

### CODE REVIEW

File: src/modules/update.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  await fetch(scoresUrl, {
+    method: 'POST',
+    body: JSON.stringify({
+      user,
+      score,
+    }),
+    headers: {
+      'Content-type': 'application/json',
+    },
+  }).then((result) => result.json());
```
Comment (codepantha) [inline src/modules/update.js:13]: - [ ] When the submit button is clicked, the POST API endpoint to submit user entries is not hit. Please fix it 🙏

File: src/modules/fetchScores.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    </table>`;
+  });
+};
+
+// fetch scores list from the API
+const fetchScores = async () => {
+  await fetch(scoresUrl)
+    .then((data) => data.json())
+    .then((scores) => showScores(scores.result));
+};
```
Comment (codepantha) [inline src/modules/fetchScores.js:21]: - [ ] When the refresh button is clicked, the GET API endpoint that fetches the latest scores list does not get hit. Please try to fetch and display the scores when the `refresh` button is clicked 🙏

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️ 

## Hi @PNdunguMaina 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @PNdunguMaina, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️
- API is working fine, good job. ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**